### PR TITLE
Build and run QueryApi

### PR DESCRIPTION
**Pull request by darunrs**
Initializes queryapi

### CODE REVIEW

**Comment by itegulov** [comment]
Where do I get the images from? I am getting:
```
Container stderr: Unable to find image 'hasura/auth:latest' locally
docker: Error response from daemon: pull access denied for hasura/auth, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
```

**Comment by darunrs** [comment]
Ah that's a great point. Let me update the readme with that info. But, you need to clone queryapi, and run docker-compose build.

**Comment by pkudinov** [comment]
@darunrs can we do that programmatically? Ideally users should not need to clone QueryAPI. Perhaps we can publish docker images of QueryAPI to DockerHub?

**Comment by darunrs** [comment]
Correct we can do things like pushing to ghcr or docker build for example. This is just the start for now. We definitely want a programatic process next. My main focus for now is getting queryapi to work. But, I'll look into it now and see if I can get that working.

**Comment by itegulov** [comment]
FYI it seems like docker compose from queryapi repo creates images prefixed with `queryapi-` (e.g. `queryapi-hasura-auth`). Is it expected that I need to retag them?

**Comment by pkudinov** [comment]
Getting this:

```2023-12-05T22:07:12.851825Z  INFO ThreadId(01) near_hat::containers::hasura_auth: starting Hasura Auth container
2023-12-05T22:07:15.780268Z ERROR ThreadId(01) testcontainers::clients::cli: Failed to start container.
Container stdout:
Container stderr: Unable to find image 'hasura/auth:latest' locally
docker: Error response from daemon: pull access denied for hasura/auth, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.```

File: README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Tooling suite for local development in NEAR ecosystem.
 
+In order to run the queryapi components, you need to clone [QueryApi](https://github.com/near/queryapi) , and then run '''docker-compose build'''. This builds local images which are pulled and used for building the dockers. 
+
+TODO: Publish queryapi images to DockerHub/GHCR to not require cloning of the repo.
```
Comment (morgsmccauley) [inline README.md]: Not needed now that you've published to dockerhub?

**Review by itegulov — APPROVED** [review]
Seems to work fine!

**Pull request merged**